test.js

```diff
@@ -62,9 +62,25 @@ callbackFunction = function(response){
   });
 };
 
+// Parse the incoming text so that each word is read as a token.
+// This will break the incoming text message into a series of tokens
+// in a list. A message would be broken as follows:
+// Input: "The quick brown fox ".
+// Output: {["The"] ["quick"] ["brown"] ["fox"]}
 // Check to see if we should send the message; this
 // is done by checking for certain keywords.
 function checkForCommand(inText){
+	
+	//If the message does not have "hey bob" (with any caps) just ignore it right away
+	if (req.body.inText.toLower.indexOf("hey bob") = -1)
```

Comment:
At this point you no longer have access to (req.body). My suggestion initially was to place this higher (I'll leave another comment up there); before tokenizing or even really beginning to process the text, we want to see if it even qualifies to be a command.